projects/js-packages/publicize-components/src/components/form/index.tsx

```diff
@@ -61,12 +61,12 @@ export default function PublicizeForm() {
 					<ShareCountInfo />
 				</>
 			) : null }
-			<PanelRow>
-				{
-					// Use IIFE make it more readable and avoid nested ternary operators.
-					( () => {
-						if ( needsUserConnection ) {
-							return (
+			{
+				// Use IIFE make it more readable and avoid nested ternary operators.
+				( () => {
+					if ( needsUserConnection ) {
+						return (
+							<PanelRow>
```

Comment:
Should we take this opportunity to extract that IIFE to a component like `connection-notice.tsx` or something similar?